stable/dex/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: dex
-version: 0.6.0
+version: 0.6.1
```

Comment:
this is not a bug fix, it is adding new functionality. I would say bump to 0.7.0. However, since this chart is in the stable folder I vote to turn that 1.0.0, as we had discussed some time ago in a charts meetings. @lachie83 any thoughts?